.github/workflows/build.yml

```
@@ -122,88 +144,63 @@ jobs:
-            bash
-      - name: Nuitka ccache
-        uses: actions/cache@v5
-        with:
-          path: ${{ github.workspace }}/.nuitka
-          key: ${{ github.job }}-ccache-alpine
-      - name: Build with nuitka
-        shell: alpine.sh {0}
-        run: make executable
+      - *install-emake
+      - name: Building executable
+        run: |
```

actionlint (1.7.12)

[error] property "nocompress" is not defined in object type {arch: string; libc: string; python: number}

(expression)